File: Prime Number/prime.py
number=int(input("Enter a number: "))
if number > 1:
    for i in range(2,int(number*0.5)+1):
        if number%i==0:
            print("not prime")
            break
    else:
        print("prime")
else:
    print("enter number greater than 1")

File: Prime Number/prime.c
#include<stdio.h>

int main(){

    int number;
    printf("Enter number: ");
    scanf("%d",&number);

    if (number>1){
        for(int i=2;i<=number/2;i++){
            if (number%i==0){
                printf("not prime");
                return 0; //Exit early is not prime
            }
        }
        printf("prime"); //if Loop completed without finding any factor
    }
    else{
        printf("enter number greater than 1");
    }
    
    return 0;
}